src/stores/auth/store.tsx

```
@@ -10,6 +10,8 @@ import { clearPasswordVerificationHash, loginRequest, storePasswordVerificationH
 import { cancelScheduledTokenRefresh, initTokenRefresh, performTokenRefresh, scheduleTokenRefresh } from '../../lib/auth/token-refresh';
 import type { AuthResponse, AuthState, LoginCredentials } from '../../lib/auth/types';
 import { type ProfileModel } from '../../lib/auth/types';
+import { cacheManager } from '@/lib/cache/cache-manager';
+import { clearCacheScope, setCacheScope } from '@/lib/cache/cache-scope';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)